[BUG] test_floor_ceil_overflow failed with Spark 3.3.0-SNAPSHOT

test_floor_ceil_overflow failed for Decimal(36,-5) and Decimal(38,-10) when running with Spark 3.3.0-SNAPSHOT.   Excerpts from the test failure:
```
[2021-11-24T09:25:00.874Z] =================================== FAILURES ===================================
[2021-11-24T09:25:00.874Z] ___________________ test_floor_ceil_overflow[Decimal(36,-5)] ___________________
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] data_gen = Decimal(36,-5)
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z]     @pytest.mark.parametrize('data_gen', [decimal_gen_36_neg5, decimal_gen_38_neg10], ids=idfn)
[2021-11-24T09:25:00.874Z]     def test_floor_ceil_overflow(data_gen):
[2021-11-24T09:25:00.874Z] >       assert_gpu_and_cpu_error(
[2021-11-24T09:25:00.874Z]             lambda spark: unary_op_df(spark, data_gen).selectExpr('floor(a)').collect(),
[2021-11-24T09:25:00.874Z]             conf=allow_negative_scale_of_decimal_conf,
[2021-11-24T09:25:00.874Z]             error_message="java.lang.ArithmeticException")
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] ../../src/main/python/arithmetic_ops_test.py:354: 
[2021-11-24T09:25:00.874Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[2021-11-24T09:25:00.874Z] ../../src/main/python/asserts.py:569: in assert_gpu_and_cpu_error
[2021-11-24T09:25:00.874Z]     assert_py4j_exception(lambda: with_cpu_session(df_fun, conf), error_message)
[2021-11-24T09:25:00.874Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] func = <function assert_gpu_and_cpu_error.<locals>.<lambda> at 0x7f12b30fa310>
[2021-11-24T09:25:00.874Z] error_message = 'java.lang.ArithmeticException'
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z]     def assert_py4j_exception(func, error_message):
[2021-11-24T09:25:00.874Z]         """
[2021-11-24T09:25:00.874Z]         Assert that a specific Java exception is thrown
[2021-11-24T09:25:00.874Z]         :param func: a function to be verified
[2021-11-24T09:25:00.874Z]         :param error_message: a string such as the one produce by java.lang.Exception.toString
[2021-11-24T09:25:00.874Z]         :return: Assertion failure if no exception matching error_message has occurred.
[2021-11-24T09:25:00.874Z]         """
[2021-11-24T09:25:00.874Z]         with pytest.raises(Py4JJavaError) as py4jError:
[2021-11-24T09:25:00.874Z]             func()
[2021-11-24T09:25:00.874Z] >       assert error_message in str(py4jError.value.java_exception)
[2021-11-24T09:25:00.874Z] E       AssertionError
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] ../../src/main/python/asserts.py:558: AssertionError
[2021-11-24T09:25:00.874Z] __________________ test_floor_ceil_overflow[Decimal(38,-10)] ___________________
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] data_gen = Decimal(38,-10)
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z]     @pytest.mark.parametrize('data_gen', [decimal_gen_36_neg5, decimal_gen_38_neg10], ids=idfn)
[2021-11-24T09:25:00.874Z]     def test_floor_ceil_overflow(data_gen):
[2021-11-24T09:25:00.874Z] >       assert_gpu_and_cpu_error(
[2021-11-24T09:25:00.874Z]             lambda spark: unary_op_df(spark, data_gen).selectExpr('floor(a)').collect(),
[2021-11-24T09:25:00.874Z]             conf=allow_negative_scale_of_decimal_conf,
[2021-11-24T09:25:00.874Z]             error_message="java.lang.ArithmeticException")
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] ../../src/main/python/arithmetic_ops_test.py:354: 
[2021-11-24T09:25:00.874Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[2021-11-24T09:25:00.874Z] ../../src/main/python/asserts.py:569: in assert_gpu_and_cpu_error
[2021-11-24T09:25:00.874Z]     assert_py4j_exception(lambda: with_cpu_session(df_fun, conf), error_message)
[2021-11-24T09:25:00.874Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] func = <function assert_gpu_and_cpu_error.<locals>.<lambda> at 0x7f12b30a94c0>
[2021-11-24T09:25:00.874Z] error_message = 'java.lang.ArithmeticException'
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z]     def assert_py4j_exception(func, error_message):
[2021-11-24T09:25:00.874Z]         """
[2021-11-24T09:25:00.874Z]         Assert that a specific Java exception is thrown
[2021-11-24T09:25:00.874Z]         :param func: a function to be verified
[2021-11-24T09:25:00.874Z]         :param error_message: a string such as the one produce by java.lang.Exception.toString
[2021-11-24T09:25:00.874Z]         :return: Assertion failure if no exception matching error_message has occurred.
[2021-11-24T09:25:00.874Z]         """
[2021-11-24T09:25:00.874Z]         with pytest.raises(Py4JJavaError) as py4jError:
[2021-11-24T09:25:00.874Z]             func()
[2021-11-24T09:25:00.874Z] >       assert error_message in str(py4jError.value.java_exception)
[2021-11-24T09:25:00.874Z] E       AssertionError
[2021-11-24T09:25:00.874Z] 
[2021-11-24T09:25:00.874Z] ../../src/main/python/asserts.py:558: AssertionError
```

--- a/integration_tests/src/main/python/arithmetic_ops_test.py
+++ b/integration_tests/src/main/python/arithmetic_ops_test.py
@@ -354,11 +354,11 @@ def test_floor_ceil_overflow(data_gen):
     assert_gpu_and_cpu_error(
         lambda spark: unary_op_df(spark, data_gen).selectExpr('floor(a)').collect(),
         conf=allow_negative_scale_of_decimal_conf,
-        error_message="java.lang.ArithmeticException")
+        error_message="ArithmeticException")
     assert_gpu_and_cpu_error(
         lambda spark: unary_op_df(spark, data_gen).selectExpr('ceil(a)').collect(),
         conf=allow_negative_scale_of_decimal_conf,
-        error_message="java.lang.ArithmeticException")
+        error_message="ArithmeticException")
 
 @pytest.mark.parametrize('data_gen', double_gens, ids=idfn)
 def test_rint(data_gen):

